DemoApps/AddressAutofill/app/src/main/java/com/mapbox/demo/autofill/SampleAppUtils.kt

```diff
@@ -34,14 +34,26 @@ fun LocationEngine.lastKnownLocation(context: Context, callback: (Point?) -> Uni
     })
 }
 
+/**
```

Comment:
These changes are not needed

Reply:
I'm getting errors if I try to remove "docs":
```
Undocumented: com.mapbox.search.ui.extensions/userDistanceTo/com.mapbox.common.location.LocationProvider#com.mapbox.geojson.Point#kotlin.Function1[kotlin.Double?,kotlin.Unit]/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/showToast/android.content.Context#kotlin.Int/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/isPermissionGranted/android.content.Context#kotlin.String/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/bitmapFromDrawableRes/android.content.Context#kotlin.Int/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/dpToPx/#kotlin.Int/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/geoIntent/#com.mapbox.geojson.Point/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/shareIntent/#com.mapbox.search.ui.view.place.SearchPlace/ (androidJvm)
Undocumented: com.mapbox.search.ui.utils.extenstion/hideKeyboard/android.view.View#/ (androidJvm)
```

Reply:
Do you get thsse error when you build project in `MapboxSearch` dir?

Note that `DemoApps` is an independent directory with standalone apps. Also note the difference in package names `com.mapbox.search.ui` - `com.mapbox.demo`